front/src/modules/timetableItem/components/Timetable/RoundTrips/TodoColumn.tsx

```diff
@@ -73,6 +73,65 @@ const TodoColumn = ({
     setItemIdToPair(itemId);
   };
 
+  const pairItems = (candidateItem: PairingItem) => {
+    if (!itemIdToPair || !setPairingItems) return;
+    // This item is always in todo
+    const matchingItem = pairingItemsById?.get(itemIdToPair);
+    const timetableItemToPair = timetableItemsWithOpsById.get(itemIdToPair);
+    const candidateItemToPair = timetableItemsWithOpsById.get(candidateItem.id);
```

Comment:
I like the naming with A and B you already used earlier, could you use it here too ?
```suggestion
    const itemA = pairingItemsById?.get(itemIdToPair);
    const timetableItemA = timetableItemsWithOpsById.get(itemIdToPair);
    const timetableItemB = timetableItemsWithOpsById.get(itemB.id);
```

Reply:
Done